src/app.js

```diff
@@ -8,11 +8,9 @@ var TelemetryUtil = require('sb_telemetry_util')
 var telemetry = new TelemetryUtil()
 var fs = require('fs')
 var configUtil = require('sb-config-util')
-var _ = require('underscore')
-var filename = path.basename(__filename)
-var utilsService = require('./service/utilsService')
-var LOG = require('sb_logger_util')
+
 const contentProvider = require('sb_content_provider_util')
+var contentMeta = require('./contentMetaFilter')
```

Comment:
why is the name contentMeta what is the use of the file please specify that

Reply:
Changed to contentMetaProvider